src/aap_eda/settings/default.py

```diff
@@ -133,14 +133,16 @@ def _get_secret_key() -> str:
 
 # JWT token lifetime
 JWT_ACCESS_TOKEN_LIFETIME_SECONDS = settings.get(
-    "JWT_ACCESS_TOKEN_LIFETIME_SECONDS", 5
+    "JWT_ACCESS_TOKEN_LIFETIME_SECONDS",
+    5,
 )
 JWT_REFRESH_TOKEN_LIFETIME_DAYS = settings.get(
-    "JWT_REFRESH_TOKEN_LIFETIME_DAYS", 2
+    "JWT_REFRESH_TOKEN_LIFETIME_DAYS",
+    2,
 )
 SIMPLE_JWT = {
     "ACCESS_TOKEN_LIFETIME": timedelta(
-        minutes=JWT_ACCESS_TOKEN_LIFETIME_SECONDS
+        seconds=JWT_ACCESS_TOKEN_LIFETIME_SECONDS,
```

Comment:
The original thought is actually `minutes=JWT_ACCESS_TOKEN_LIFETIME_MINUTES`

Reply:
changed.